packages/schematics/angular/library/files/__projectRoot__/tsconfig.lib.json

```diff
@@ -24,7 +24,8 @@
     "fullTemplateTypeCheck": true,
     "strictInjectionParameters": true,
     "flatModuleId": "AUTOGENERATED",
-    "flatModuleOutFile": "AUTOGENERATED"
+    "flatModuleOutFile": "AUTOGENERATED",
+    "enableResourceInlining": true
```

Comment:
Is this required for the watch mode? If so we might need to warn users of older versions about it.

Reply:
Yes this is required for V4. And we removed a lot of internal logic to inline resources.

Reply:
Ok, then I think we'll need to check this as well when asserting compatibility.

Reply:
For v3, this flag might cause some unwanted behaviour I think.

Reply:
Yeah makes sense. 

Maybe we can fully remove these values and let ng-packagr take care of them? I think we ended up adding them manually to get around a tsconfig merging bug in ng-packagr. It might not be present anymore.

If they are still needed then we need to do the health check both ways then: that they are not present when using ng-packagr v3, and that they are when using v4.

Reply:
Actually I gave this a bit more thought. And I think. IN V3, this might not have any `sideEffects` as there is a previous step to transforms the components.

I have updated the v3 branch to include this flag, and from the Unit Tests all seem fine
https://github.com/dherges/ng-packagr/pull/989

Reply:
Awesome!